backend/api/src/test/kotlin/software/shonk/adapters/incoming/ShorkInterpreterControllerV1IT.kt

```diff
@@ -748,4 +749,91 @@ class ShorkInterpreterControllerV1IT : AbstractControllerTest() {
             assertEquals(HttpStatusCode.NotFound, result.status)
         }
     }
+
+    @Nested
+    inner class PostLobbySettings {
+        @Test
+        fun `update settings for an existing lobby`() = runTest {
+            val clientLobby =
+                client.post("/api/v1/lobby") {
+                    contentType(ContentType.Application.Json)
+                    setBody("{\"playerName\":\"playerA\"}")
+                }
+            val lobbyId =
+                Json.parseToJsonElement(clientLobby.bodyAsText())
+                    .jsonObject["lobbyId"]!!
+                    .jsonPrimitive
+                    .content
+            val updatedSettings =
+                InterpreterSettings(
+                    coreSize = 2048,
+                    instructionLimit = 500,
+                    initialInstruction = "ADD",
+                    maximumTicks = 100000,
+                    maximumProcessesPerPlayer = 16,
+                    readDistance = 100,
+                    writeDistance = 100,
+                    minimumSeparation = 50,
+                    separation = 50,
+                    randomSeparation = true,
+                )
+
+            val updateResponse =
+                client.post("/api/v1/lobby/$lobbyId/settings") {
+                    contentType(ContentType.Application.Json)
+                    setBody(Json.encodeToString(updatedSettings))
+                }
+
+            assertEquals(HttpStatusCode.OK, updateResponse.status)
+        }
```

Comment:
Can you extend the test or add another one where you first set the lobby settings and then request them back to check if they were actually set?